server/metrics/metrics.go

```diff
@@ -38,6 +38,9 @@ const (
 	// CRIOImageLayerReuseKey is the key for the CRI-O image layer reuse metrics.
 	CRIOImageLayerReuseKey = "crio_image_layer_reuse"
 
+	// CRIOContainersOOMKey is the key for the CRI-O container out of memory metrics.
+	CRIOContainersOOMKey = "crio_containers_oom"
```

Comment:
My suggestion would be to use `_total` suffix here, as per best practices:

```suggestion
	CRIOContainersOOMKey = "crio_containers_oom_total"
```

Reply:
Fixed as suggested :+1: